andela/titan-dev-ah-frontend
# #165352853  Read Article

#### What does this PR do?
- Read article 
#### Description of Task to be completed?
- Fetch the article from a database 
- Transform article body into html 
- Retrieve  articles that exist to the right side of a single article
#### How should this be manually tested?
- Clone the repository and check out the branch `ft-read article-165352853`
- Run `yarn` to install all dependencies
- Run `yarn test` to run all tests 
- Start your backend server 
- Create an article using postman client tool by consuming `POST: {backEndUrl}/api/v1/articles`
- Get the `slug` of the article created
- Start the front-end server by running `yarn start`
- Navigate to `localhost:8080/articles/slug`
- You should see the article you created, with other articles in your database on the right side of the main article. 

#### What are the relevant pivotal tracker stories?
#165352853
